### DISCUSSION TITLE
Is it possible to add plugin for react hook version of OS?

### DISCUSSION THREAD

**Discussion by Enkratia** [KingSora/OverlayScrollbars#633] [Q&A] [answered]
I don't see any possibility to add plugin here: 
`useOverlayScrollbars({ options: osOptions, defer: true });

Question #2: OS react hook doesn't have option to set element, right?

**Comment by KingSora** [comment, marked as answer]
Good day @Enkratia :)

Adding plugins is not something thats specific to any framework. Plugins are global and should not be added in a component, instead they should be added in your entry point or somewhere in the global scope. You can add plugins like this:

```js
import { OverlayScrollbars } from 'overlayscrollbars';

OverlayScrollbars.plugin(MyPlugin); 
```

The react hook has the possibility to attach OverlayScrollbars to any element. You can find an example of this in the README of the react version: https://github.com/KingSora/OverlayScrollbars/tree/master/packages/overlayscrollbars-react#hook

**Reply by Enkratia** [reply]
Good day @KingSora. Thank you
